src/main/archetype/ui.apps.structure/pom.xml

```diff
@@ -49,6 +49,7 @@
 
                         <!-- DAM folder root, will be created via repoinit -->
                         <filter><root>/content/dam/${appId}</root></filter>
+                        <filter><root>/content/dam/${appId}/jcr:content</root></filter>
```

Comment:
This additional filter is redundant. Are you sure this is effective at all?

Reply:
No, you're right, it's absolutely useless 🤦 

Since you added the new `MERGE_PROPERTIES` mode in https://issues.apache.org/jira/browse/JCRVLT-255 I think we can use that.

The whole idea was to avoid having a broad filter on the `/content/dam/${appId}` node as it was causing issues during deployments on Cloud Service.

Reply:
Alternatively, we could add the `/content/dam/${appId}` filter and `jcr:content` node definition only for 6.5 projects.